src/home_robot/home_robot/utils/geometry/angles.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+
+PI2 = 2 * np.pi
+
+
+def angle_difference(angle1: float, angle2: float):
+    """angle difference"""
+    angle1 = angle1 % PI2
+    angle2 = angle2 % PI2
+    return np.abs(angle1 - angle2)
+
+
+def interpolate_angles(start_angle, end_angle, step_size: float = 0.1):
+    start_angle = start_angle % PI2
+    end_angle = end_angle % PI2
+    interpolated_angle = start_angle + (step_size * np.sign(end_angle - start_angle))
+    return interpolated_angle % PI2
```

Comment:
That may not work in right direction for cases like start_angle = 0.1 and end_angle = 2 * np.pi - 0.1. The function would interpolate in the positive direction, moving away from end_angle rather than towards it.
 
```suggestion
def interpolate_angles(start_angle: float, end_angle: float, step_size: float = 0.1) -> float:
    """Interpolate between two angles in radians with a given step size."""
    start_angle = start_angle % PI2
    end_angle = end_angle % PI2
    direction = np.sign((end_angle - start_angle + PI2) % PI2 - np.pi)
    interpolated_angle = start_angle + (step_size * direction)
    return interpolated_angle % PI2
```

Reply:
We should add a unit test to code this

Reply:
will do that before committing the suggestion